packages/pieces/community/ing-australia/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@activepieces/piece-ing-australia",
+  "version": "0.0.1",
+  "dependencies": {
+    "ing-au-login": "2.0.7",
+    "puppeteer": "22.13.1",
```

Comment:
Does ActivePieces automatically install these dependencies? In the development environment, I need to manually install them.